samples/sample_monthly_update.md

```diff
@@ -0,0 +1,70 @@
+# Sample Update
+
+We ask our companies to send us regular updates so that we can stay up to date with the company, and help out as necessary. To help us help you, here is a sample of what we need to see.
+
+We ask that you spend at most 15-30 minutes filling out this information, as you will want systems in place to easily track this data.
+
+## Outline
+
+* **Our Victories**
+* **Our Struggles**
+* **Key Metrics**
+* **How You Can Help**
+* **Other Notes**
+
+---
+
+## Teacher Tools Inc. - May Update
+
+We're excited to move into June after a great school year. We've had our third consequetive month of growth and we've hired two new developers. Unfortunately our spending is a little higher than expected and we're adjusting our plans.
+
+### Our Victories
+
+1. We've grown our customer base in Ontario and Québec, and we're on track to meet our next milestones.
+2. Our team spent the last iteration on adding the new dashboard and it shipped on June 8th.
+
+### Our Struggles
+
+1. We're spending an aggressive amount on ads in order to acquire customers.
+2. We had mixed feedback on our dashboard so far. We were expecting a more positive response.
+3. Our dashboard took three weeks to ship instead of one. We underestimated how complex it would be.
+
+### Key Metrics
+
+#### Business and Financial
```

Comment:
This might be a little heavy for what we need, but they're good metrics to know. Recommendations on a trimmer, more to-the-point list? Anything missing?